src/Pages/ProductList/ProductSpec/ProductSpec.js

```diff
@@ -1,7 +1,241 @@
+    },
+    {
+      id: 2,
+      title: 'Reviews',
+      description: <h2>Reviews</h2>,
+    },
+  ];
+
+  //myBag modal part
+  const showMyBag = () => {
+    size && setIsClosedBoxModal(!isClosedBoxModal);
+  };
+
+  //wishList modal part
+  const handleWishListBtn = () => {
+    setIsAddedWishList(!isAddedWishList);
+  };
+
+  const showWishList = () => {
+    handleWishListBtn();
+    !isAddedWishList && setIsShowedWishModal(!isShowedWishModal);
+  };
+
+  //Thumnail, Main img part
+  const handleClickedImg = e => {
+    const clickedIdx = e.target.className.split(' ')[0];
+    setClickedImg(clickedIdx);
+  };
+
+  const checkClickedImg = i => {
+    return i === clickedImg;
+  };
+
+  //Detail part
+  const scrollToDetail = () => {
+    detailRef.current.scrollIntoView({ behavior: 'smooth' });
+  };
+
+  //Quantitiy part
+  const plusCount = () => {
+    setCount(count + 1);
+  };
+
+  const minusCount = () => {
+    setCount(count <= 0 ? 0 : count - 1);
+  };
+
+  //Store clothing's size value
+  const handleSize = e => {
+    const { value } = e.target;
+
+    setSize(value);
+  };
+
+  //Description part
+  const handleCheckedList = e => {
+    setCheckedList(e.target.className);
+  };
+
+  const checkList = i => {
+    return i === checkedList;
+  };
+
+  return (
+    <div className="productSpec">
+      {isShowedWishModal && (
+        <WishListModal
+          isShowedWishModal={isShowedWishModal}
+          setIsShowedWishModal={setIsShowedWishModal}
+        />
+      )}
+      <MyBagModal isClosed={isClosedBoxModal} showMyBag={showMyBag} />
+      <div className="spec row">
+        <div className="imgContainer">
+          <div className="thumnails">
+            {image_urls.map((img, i) => {
+              return (
+                <img
+                  key={i}
+                  className={
+                    checkClickedImg(`${i}`)
+                      ? `${i} thumnail clicked`
+                      : `${i} thumnail`
+                  }
+                  alt="thumnail"
+                  src={img}
+                  onClick={handleClickedImg}
+                />
+              );
+            })}
+          </div>
+          <div className="mainImg">
+            <img alt="main-img" src={image_urls[clickedImg]} />
+          </div>
+        </div>
+        <div className="orderContainer">
+          <div className="orderList">
+            <h2>{name}</h2>
+            <h2>${price}</h2>
+            <div className="detail">
+              <p>{detail}</p>
+              <span onClick={scrollToDetail}>View Details</span>
+            </div>
+
+            <div className="sizeOptions">
+              <h3> Size </h3>
+              {sizeList.map(list => {
```

Comment:
파라미터 자리에서 바로 구조분해 할당 가능합니다. 데이터량이 많지 않다면 파라미터에서 바로 구조분해 해주는게 떠 깔끔해 보일 것 같아요:)
```suggestion
              {sizeList.map(({id, value, name}) => {
```

Reply:
오 이런 방법이! 감사합니다! 수정하였습니다!